Microsoft.Dynamics365.UIAutomation.Api.UCI/WebClient.cs

```diff
@@ -1257,8 +1257,16 @@ internal BrowserCommandResult<bool> OpenRecord(int index, int thinkTime = Consta
                     {
                         if (currentindex == index)
                         {
-                            var tag = checkRecord ? "div" : "a";
-                            row.FindElement(By.TagName(tag)).Click();
+                            var tag = "div";
+                            if (checkRecord)
+                            {
+                                row.FindElement(By.TagName(tag)).Click();
+                            }
+                            else
+                            {
+                                driver.DoubleClick(row.FindElement(By.TagName(tag)));

```

Comment:
This doble click is not working for my. Tested in Chrome v9.1 online. 
test sample: UCITestAccountAddAppointment -> xrmApp.Grid.OpenRecord(0);

I understand the intention but I will roll it back, until a better approach is found.